packages/playground/website/playwright/e2e/deployment.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { test, expect } from '../playground-fixtures.ts';
+import { spawn } from 'child_process';
+
+const host = 'localhost';
+const port = 7999;
+const url = `http://${host}:${port}`;
+const maxDiffPixels = 2000;
+
+type Server = {
+	kill: () => Promise<void>;
+};
+
+const startServer = async (page, path: string): Promise<Server> => {
+	const cp = spawn('python3', ['-m', 'http.server', port.toString()], {
```

Comment:
What if we used `php -S` here instead as this project has more of a relationship with PHP than Python. Maybe fewer would need to install a Python dependency.

Alternately, is there a reason we shouldn't just run a node-based HTTP server like [`serve`](https://www.npmjs.com/package/serve)?

Reply:
I was having trouble getting the tests to run because of ENOENT errors when `python3` couldn't be found. It must have been some kind of pathing issue because `python3` is in my PATH. The same thing happened for `php`.

So I went ahead and switched to an inline Node.js-based HTTP server with https://github.com/WordPress/wordpress-playground/pull/1822/commits/fadf6e97b6cbdb868006ba3410e68766726c3f2f. It seems like this will be more reliable for more folks than using Python or PHP installed separately. Please feel free to revert if needed though.